pytest_tests/tests/payment/test_deposit_withdrawal.py

```diff
@@ -48,13 +67,10 @@ def test_deposit_withdrawal(self, neofs_env_with_mainchain: NeoFSEnv):
                 wallet_config=neo_go_wallet_config,
                 await_=True,
             )
-            assert get_wallet_balance(neofs_env, neo_go, wallet, neo_go_wallet_config) <= 900, (
-                "Wallet balance is not correct after deposit"
-            )
-            neofs_cli = neofs_env.neofs_cli(cli_wallet_config)
-            assert get_neofs_balance(neofs_env, neofs_cli, wallet) == 100, (
-                "Wallet balance in neofs is not correct after deposit"
+            wait_for_correct_wallet_balance(
+                neofs_env, wallet, neo_go, neo_go_wallet_config, lambda balance: balance <= 900
             )
+            wait_for_correct_neofs_balance(neofs_env, wallet, cli_wallet_config, lambda balance: balance == 100)
```

Comment:
This however is different, it's NeoFS node replying and it can use a different RPC or cache things, so here some waiting is appropriate.